features/lwipstack/lwipopts.h

```diff
@@ -84,7 +84,7 @@
 #define SYS_LIGHTWEIGHT_PROT        1
 
 #ifndef LWIP_RAW
-#define LWIP_RAW                    0
+#define LWIP_RAW                    1
```

Comment:
This should be configurable via JSON, like TCP. Enabling raw socket support will increase build size of lwIP, and most people will not need it.

I think it should be off by default, but maybe that could be debated.

Reply:
yes I agree.